services/thermal_mgr/thermal_mgr.c

```diff
@@ -42,19 +42,34 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 }
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
-  /* Send an event to the thermal manager queue */
-
+  if (xQueueSend(thermalMgrQueueHandle, event, portMAX_DELAY) != pdPASS) {
```

Comment:
This function is called in an ISR and we should never be blocking in an ISR, so we shouldn't be using `portMAX_DELAY` since this means we will block indefinitely until there is space in the queue